index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
+        </div>
+        <!-- CARD 2 -->
+        <div class="card-2">
+            <img src="image/Snapshoot Portfolio2.png" alt="image1" >
+            <h1 class="tonic-2">Tonic</h1>
+            <div class="canop-2">
+                <p class="can-info-2">CANOPY</p>
+                <div class="point-2"></div>
+                <p class="back-2">Back End Dev</p>
+                <div class="point-2"></div>
+                <p class="back-2">2015</p>
+            </div>
+            <div class="languages-2">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-2">See project</button>
+        </div>
+        <!-- CARD 3 -->
+        <div class="card-3">
+            <img src="image/Snapshoot Portfolio3.png" alt="image1" >
+            <h1 class="tonic-3">Tonic</h1>
+            <div class="canop-3">
+                <p class="can-info-3">CANOPY</p>
+                <div class="point-3"></div>
+                <p class="back-3">Back End Dev</p>
+                <div class="point-3"></div>
+                <p class="back-3">2015</p>
+            </div>
+            <div class="languages-3">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-3">See project</button>
+        </div>
+        <!-- CARD 4 -->
+        <div class="card-4">
+            <img src="image/Snapshoot Portfolio4.png" alt="image1" >
+            <h1 class="tonic-4">Tonic</h1>
+            <div class="canop-4">
+                <p class="can-info-4">CANOPY</p>
+                <div class="point-4"></div>
+                <p class="back-4">Back End Dev</p>
+                <div class="point-4"></div>
+                <p class="back-4">2015</p>
+            </div>
+            <div class="languages-4">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see-4">See project</button>
+        </div>
+    </section>
+    <!-- ABOUT SECTION -->
+    <section class="myself">
+        <h1 class="me">About Myself</h1>
+        <p class="para-detail">
+            Hello I'm a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <p class="conn-5">LET'S CONNECT</p>
+        <ul class="social-5">
+            <li><a href="#"><img src="image/Vector (12).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Linkedin icon.png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (13).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (15).png" alt=""></a></li>
+            <li><a href="#"><img src="image/Vector (11).png" alt=""></a></li>
+        </ul>
```

Comment:
- [ ] Great job on adding the social icons using a list! The use of an unordered list (`<ul>`) is a good approach for grouping the icons. However, I would recommend using flexbox to align the items within the list. This will give you more control over the alignment and spacing of the icons. You can apply the `display: flex` property to the parent `<ul>` element and use flexbox properties like `justify-content` and `align-items` to align the icons as desired.

Keep up the good work, and don't hesitate to ask if you have any questions! 👍🌟